semantic/flatbuffers_gen.go

```diff
@@ -525,26 +525,26 @@ func (rcv *OptionStatement) FromBuf(fb *fbsemantic.OptionStatement) error {
 	return nil
 }
 
-func (rcv *Package) FromBuf(fb *fbsemantic.Package) error {
+func (p *Package) FromBuf(fb *fbsemantic.Package) error {
```

Comment:
This file is generated code, if there are no real changes here, it's probably better to roll them back so as not to fail `make checkgenerate`.